### PR TITLE
DS-762 | @ericbakenhus | Use normal function for deploy handler

### PR DESCRIPTION
**Pull request by ericbakenhus**
# READY FOR REVIEW

# Summary
- Switch deploy handler to normal function

# Review By (Date)
- ASAP/Retro

# Criticality
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code

# Associated Issues and/or People
- DS-762

### CODE REVIEW

File: netlify/functions/handle-deploy.ts [hunk 1]
```diff
@@ -89,22 +89,23 @@ export default async (req: Request) => {
...
-      return;
+      return new Response('Accepted', { status: 202 });
     }
     
   } catch (err) {
     console.error('Error during deploy function: ', err);
   }
 
   console.log('=== END Deploy Background Function ===');
+  return new Response('Accepted', { status: 202 });
```
Comment (ericbakenhus) [inline netlify/functions/handle-deploy.ts:108]: Just going to return 202 for everything to stay consistent. 🤷

**Comment by ericbakenhus** [comment]
I'm going to merge this and test in dev so I don't have to spend an hour messing with temporary Netlify env vars and Storyblok config changes. 🤠

Please feel free to test there as well.

**Comment by ericbakenhus** [comment]
Looks good:

Screenshot 2024-06-19 at 9 00 35 PM

File: netlify/functions/handle-deploy.ts (hunk 1 above)
Comment (sherakama) [reply netlify/functions/handle-deploy.ts:108]: Works for me. Thanks.

**Review by sherakama — COMMENTED** [review]
Looks good.

**Pull request merged**